.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+
+repos:
+- repo: https://github.com/psf/black
+  rev: 20.8b1
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.3.0
+  hooks:
+  - id: check-added-large-files
+    args: ['--maxkb=1000']
+  - id: mixed-line-ending
+  - id: trailing-whitespace
+  - id: check-merge-conflict
+  - id: check-case-conflict
+  - id: check-symlinks
+  - id: check-yaml
+  - id: requirements-txt-fixer
+  - id: debug-statements
+  - id: end-of-file-fixer
+  - id: fix-encoding-pragma
+- repo: https://github.com/mgedmin/check-manifest
+  rev: "0.45"
+  hooks:
+  - id: check-manifest
+    stages: [manual]
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.790
+  hooks:
+  - id: mypy
+    files: src
+    additional_dependencies: [attrs==19.3.0]
```

Comment:
I don't think we need this? scikit-hep doesn't use attires? This should be a list of the minimal requirement to do static type checking (usually just the packages that support type checking)

Actually, more fundamentally, this is only looking at files in `/src`, which there are none, so I'd probably remove this check until we support some type checking (and then the files setting would need to be correct). Only useful if there's some code to check.

Reply:
Fair enough, I will remove these lines.

Reply:
I had simply copied and pasted the file from Particle, and not everything was relevant. Fair enough. Thanks.